configs/config.go

```diff
@@ -104,4 +138,7 @@ type Config struct {
 	// SystemProperties is a map of properties and their values. It is the equivalent of using
 	// sysctl -w my.property.name value in Linux.
 	SystemProperties map[string]string `json:"system_properties"`
+
+	// SysCalls specify the system calls to keep when executing the process inside the container
```

Comment:
nit: system calls to keep is confusing, since it depends on the action too. We should probably also mention that the default is allow everything.